### PR TITLE
Working poc

### PR DESCRIPTION
**Pull request by Skallwar**
image

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -7,4 +7,7 @@ edition = "2021"
 
 [dependencies]
 yew = "0.19"
-jinko = { git = "https://github.com/jinko-core/jinko", default-features = false }
+#jinko = { git = "https://github.com/jinko-core/jinko", default-features = false }
+jinko = { path = "../jinko", default-features = false }
```
Comment (CohenArthur) [inline Cargo.toml]: I don't know if that change should be part of this PR :D

File: src/main.rs [hunk 2]
```diff
@@ -34,17 +34,30 @@ impl Component for Model {
...
-                    Some(content) => self.input = content,
+            Msg::Update(event) => {
+                match event.input_type().as_str() {
+                    "deleteContentBackward" => {
+                        self.input.pop();
+                    }
+                    "insertText" => {
+                        event.data().as_ref().map(|data| self.input.push_str(data));
+                    }
+                    &_ => unimplemented!("\"{}\" is not handled", event.input_type().as_str()),
```
Comment (CohenArthur) [inline src/main.rs]: I assume there's no way of matching directly on the values returned by `input_type()` instead of converting them to strings?

**Review by CohenArthur — COMMENTED** [review]
Nice! Thanks a lot for working on this :)

File: src/main.rs (hunk 2 above)
Comment (Skallwar) [reply src/main.rs]: No, I didn't find https://docs.rs/web-sys/latest/web_sys/struct.InputEvent.html#method.input_type

File: Cargo.toml (hunk 1 above)
Comment (Skallwar) [reply Cargo.toml]: Yup, should be fixed now

**Comment by CohenArthur** [comment]
Sorry for not merging this sooner. I just tried it and it does not seem to do anything @Skallwar ?

**Review by CohenArthur — APPROVED** [review]
The code looks good to me otherwise :)

**Comment by Skallwar** [comment]
> Sorry for not merging this sooner. I just tried it and it does not seem to do anything @Skallwar ?

That's weird it's working for me. I took the screenshot yesterday

File: Cargo.toml (hunk 1 above)
Comment (Skallwar) [reply Cargo.toml]: Fixed

File: src/main.rs (hunk 2 above)
Comment (Skallwar) [reply src/main.rs]: The value returned by `input_type()` is a `String`

**Comment by Skallwar** [comment]
Oh it looks like for some reason the first input is not used. Reloading the page once fixes the issue. I will investigate

**Comment by Skallwar** [comment]
I changed some stuff, tell me if it's better now

**Comment by CohenArthur** [comment]
Yes! :D Looks super cool, thank you :D

**Review by CohenArthur — APPROVED** [review]

**Pull request merged**